Orange/preprocess/impute.py

```diff
@@ -81,8 +81,7 @@ class DropInstances(BaseImputeMethod):
     description = ""
 
     def __call__(self, data, variable):
-        index = data.domain.index(variable)
-        return numpy.isnan(data[:, index]).reshape(-1)
+        return numpy.isnan([list(row) for _, row in data[variable].iterrows()]).reshape(-1)
```

Comment:
Haven't measured it, but I suppose not much slower than it was before. `numpy.isnan(data[:, index])` also did a full iteration over data (via `Sequence.__iter__()` via `table.__getitem__(i)`).

In pandas, this will be: `return data[variable].isnull()`.

I'll fix it up to use `Table.get_column_view()` interim.